requirements.txt

```diff
@@ -1,3 +1,9 @@
-torch==1.7.1
-clip @ git+https://github.com/openai/CLIP.git@cfcffb90e69f37bf2ff1e988237a0fbe41f33c04
 jina==2.0.0rc6
+resampy==0.2.2
+tensorflow_cpu==2.2.0
+tf_slim==1.1.0
+six==1.15.0
+soundfile==0.10.3.post1
+librosa==0.8.0
+click==7.1.2
```

Comment:
why click needed?

Reply:
right, click should not be needed

Reply:
or pandas either. If this is needed for testing add it in the local requirements file of the test folder